src/Features/LanguageServer/Protocol/CustomProtocol/FindUsagesLSPContext.cs

```diff
@@ -114,6 +116,14 @@ public override async Task OnReferenceFoundAsync(SourceReferenceItem reference)
                     return;
                 }
 
+                // If the definition hasn't been reported yet, add it to our list of references to report.
+                var definitionItem = _definitionsWithoutReferences.Where(d => d.DefinitionId == definitionId).FirstOrDefault();
```

Comment:
this can then just be `if (_definitionsWithoutReference.TryGetValue(definitionId, out var definition))` ...